drivers/adc/ad9250/ad9250.c

```diff
@@ -413,28 +414,18 @@ int32_t ad9250_is_shadow_register(int32_t register_address)
  *			    01 � power-down;
  *			    10 - standby.
  *
- * @return Returns negative error code or the set power mode.
+ * @return Returns negative error code or 0 in case of success.
  *******************************************************************************/
 int32_t ad9250_chip_pwr_mode(struct ad9250_dev *dev,
 			     int32_t mode)
 {
-	uint32_t ret = 0;
-
 	if((mode >= 0) && (mode < 3)) {
-		ret = ad9250_set_bits_to_reg(dev,
-					     AD9250_REG_PDWN,
-					     AD9250_PDWN_CHIP(mode),
-					     AD9250_PDWN_CHIP(0x3));
-	} else {
-		ret = ad9250_read(dev,
-				  AD9250_REG_PDWN);
-		if(ret != SUCCESS) {
-			return ret;
-		}
-		return (ret & AD9250_PDWN_CHIP(0x3)) >> 0;
-	}
-
-	return ret;
+		return (ad9250_set_bits_to_reg(dev,
+					       AD9250_REG_PDWN,
+					       AD9250_PDWN_CHIP(mode),
+					       AD9250_PDWN_CHIP(0x3)));
+	} else
```

Comment:
nitpick: else still needs brackets here

Reply:
Better to remove brackets from if actually.

Reply:
Done